### PR TITLE
fix(cloudflare): bind `getRandomValues` and `randomUUID` to the crypto object

### PR DESCRIPTION
**Pull request by petebacondarwin**
Resolves https://github.com/unjs/unenv/issues/306

### CODE REVIEW

**Review by anonrig — APPROVED** [review]

File: src/runtime/node/crypto/$cloudflare.ts [hunk 1]
```diff
@@ -108,8 +108,10 @@ export const {
 
 export const webcrypto = {
   CryptoKey: unenvCryptoWebcrypto.CryptoKey,
-  getRandomValues: workerdCrypto.webcrypto.getRandomValues,
-  randomUUID: workerdCrypto.webcrypto.randomUUID,
+  getRandomValues: workerdCrypto.webcrypto.getRandomValues.bind(
+    workerdCrypto.webcrypto,
+  ),
+  randomUUID: (...args) => randomUUID(...args).bind(workerdCrypto.webcrypto),
```
Comment (jasnell) [inline src/runtime/node/crypto/$cloudflare.ts:114]:
```suggestion
  randomUUID: workerdCrypto.randomUUID.bind(workerdCrypto.webcrypto),
```

File: src/runtime/node/crypto/$cloudflare.ts [hunk 2]
```diff
@@ -108,8 +108,10 @@ export const {
 
 export const webcrypto = {
   CryptoKey: unenvCryptoWebcrypto.CryptoKey,
-  getRandomValues: workerdCrypto.webcrypto.getRandomValues,
-  randomUUID: workerdCrypto.webcrypto.randomUUID,
+  getRandomValues: workerdCrypto.webcrypto.getRandomValues.bind(
+    workerdCrypto.webcrypto,
+  ),
```
Comment (jasnell) [inline src/runtime/node/crypto/$cloudflare.ts:113]:
```suggestion
  getRandomValues: workerdCrypto.webcrypto.getRandomValues.bind(workerdCrypto.webcrypto),
```

Comment (petebacondarwin) [reply src/runtime/node/crypto/$cloudflare.ts:113]: This one is enforced by prettier

**Review by pi0 — APPROVED** [review]

**Comment by petebacondarwin** [comment]
Actually this is not correct from some local testing. Please don't merge

**Comment by petebacondarwin** [comment]
Doh!

**Comment by pi0** [comment]
sorry for merge @petebacondarwin do you want to revert?

**Comment by petebacondarwin** [comment]
No need to revert. This doesn't break anything.
But it doesn't fix the problem either.
I'll put up a new PR tomorrow

**Pull request merged**